client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -41,8 +43,14 @@ const Email = React.createClass( {
 	},
 
 	render() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		const classes = classNames( 'domain-management-email', {
+			'is-wide-layout': showPlanFeatures
```

Comment:
Hmm, it does `margin-top: 23px` but in `plan-actions`. However, that won't be used in plan features so I don't think we should add this to `Main`. Maybe we could hack this as a class but `is-wide-layout` as a prop to `Main`. What do you think?